lib/Cake/Model/Model.php

```diff
@@ -2895,6 +2895,10 @@ public function exists($id = null) {
 			return false;
 		}
 
+		if ($this->useTable === false) {
+			return true;
```

Comment:
Thats fair enough. Was just a suggestion, I'll revert it back to returning false :)